src/main/java/io/github/thebusybiscuit/exoticgarden/listeners/PlantsListener.java

```diff
@@ -395,6 +397,28 @@ public void onBonemealPlant(BlockFertilizeEvent e) {
         }
     }
 
+    @EventHandler(priority = EventPriority.HIGHEST, ignoreCancelled = true)
+    public void onDispenserBonemealPlant(BlockDispenseEvent e) {
+        ItemStack bonemeal = e.getItem();
```

Comment:
This shouldn't really be named `bonemeal` here. We don't know if it is bone meal yet.